src/components/TasksTable/index.jsx

```diff
@@ -254,6 +315,46 @@ export default class TasksTable extends Component {
             onFilterClick={this.handleFilterToggle}
           />
         </div>
+        <Drawer
+          anchor="right"
+          open={drawerOpen}
+          onClose={this.handleDrawerClose}>
+          <div className={classes.drawer}>
+            <div className={classes.drawerHeader}>
+              <IconButton onClick={this.handleDrawerClose}>
+                <CloseIcon />
+              </IconButton>
+            </div>
+            <List>
+              <ListItem>
+                {drawerItem &&
```

Comment:
Instead of checking for `drawerItem` in every `ListItem`, you can do

```jsx
{
  drawerItem && (
    <List>
      <ListItem>
        {drawerItem.summary}
      </ListItem>
      // ...
    </List>
  )
}

```